src/properties.jl

```diff
@@ -207,7 +211,7 @@ end
 
 set the specified NIDAQ property to value
 """
-function setproperty!(t::Task, channel::String, property::String, value)
+function Base.setproperty!(t::Task, channel::String, property::String, value)
```

Comment:
shouldn't one need to `import Base: setproperty!` for this to work?

Reply:
no it is not needed to import Base. Therefore, I would leave it as it is.